modules/deployments-ab-testing-lb.adoc

```diff
@@ -54,6 +54,11 @@ Browse to the application at `ab-example-a.<project>.<router_domain>` to verify
 
 . When you deploy the route, the router balances the traffic according to the `weights` specified for the services. At this point, there is a single service with default `weight=1` so all requests go to it. Adding the other service as an `alternateBackends` and adjusting the `weights` brings the A/B setup to life. This can be done by the `oc set route-backends` command or by editing the route.
 +
+[NOTE]
+====
+When using `alternateBackends` also use the `roundrobin` load balancing strategy to ensure requests are distributed as expected to the services based on weight. `roundrobin` can be set for a route using a link:https://docs.openshift.com/container-platform/4.13/networking/routes/route-configuration.html#nw-route-specific-annotations_route-configuration[route annotation], or for the router in general using an environment variable.
```

Comment:
```
or for the router in general using an environment variable.
```
In OCP 4.x the router deployment is managed by ingress operator so user cannot update environment variable directly, please remove the statement.

Other looks good. Thanks